File: src/main/java/com/example/kontak/KontakApplication.java
package com.example.kontak;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;

@SpringBootApplication
public class KontakApplication extends SpringBootServletInitializer {


	@Override
	protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
		return application.sources(KontakApplication.class);
	}

	public static void main(String[] args) {
		SpringApplication.run(KontakApplication.class, args);
	}

	
}

File: app/src/main/java/com/example/kontak/KontakApplication.kt
package com.example.kontak

import android.app.Application
import com.example.kontak.repository.AppContainer
import com.example.kontak.repository.KontakContainer

class KontakApplication : Application() {
    lateinit var container: AppContainer
    override fun onCreate() {
        super.onCreate()
        container = KontakContainer()
    }
}